### PR TITLE
feat(minor): add `--printTimings` option to the `chat` CLI command

### PR DESCRIPTION
**Pull request by JonHolman**
### Description of change

Add a --printTimings option to the CLI

### Pull-Request Checklist

- [X] Code is up-to-date with the `master` branch
- [X] `npm run format` to apply eslint formatting
- [X] `npm run test` passes with this change
- [X] This pull request links relevant issues as `Fixes #0000`
- [ ] There are new or updated unit tests validating the change
- [ ] Documentation has been updated to reflect this change
- [X] The new commits and pull request title follow conventions explained in [pull request guidelines](https://withcatai.github.io/node-llama-cpp/guide/contributing) (PRs that do not follow this convention will not be merged)

### CODE REVIEW

**Comment by giladgd** [comment]
@JonHolman Can you please add screenshots of how it looks like after your change?

File: src/utils/getBin.ts [hunk 1]
```diff
@@ -138,6 +138,7 @@ export type LLAMAContext = {
     tokenNl(): number,
     getContextSize(): number
     getTokenString(token: number): string
+    printTimings(): undefined
```
Comment (giladgd) [inline src/utils/getBin.ts:141]:
```suggestion
    printTimings(): void
```

**Review by giladgd — APPROVED** [review]
LGTM

**Comment by JonHolman** [comment]
> @JonHolman Thanks for the PR! Can you please add screenshots of what it looks like after your change?

Screenshot 2024-01-20 at 4 28 00 PM

**Pull request merged**